### ISSUE TITLE
Detect If Authentication Fails

### ISSUE THREAD

**Issue by hydrian** [hydrian/pfsense-backups#16]
Currently the script doesn't check to see if the authentication is successful. It gives you a false positive and tries to continue. 

Add logic to detect bad authentication.

**Comment by hydrian** [comment]
Added enhancement with 5ebdb150862a0652c72d59e108f190c228d40fbd

**Comment by hydrian** [comment]
It may not be compatible in non-English locales. I'm looking for language agnostic detection method.

**Comment by khorsmann** [comment]
It should be possible to send the client language as param via wget (or curl) to the pfsense gui. So you can be sure that you get always english as response for the login process. Chrome F12 developer tools should help you out (I am currently out of the office without an testing system).

**Comment by hydrian** [comment]
That's true. I'll play with it.

**Comment by hydrian** [comment]
I played with the Accept-Language header for a bit. I don't think the Pfsense admin console honors this. 

In the admin web pages them themselves you can force the language to be a specific language. So the developers just assume that's right for all admin clients requesting the web page.

**Comment by khorsmann** [comment]
Then an comment in the Readme configbackup user should have language settings english to dedect wrong password should be fine.

Better would be if pfsense itself used http return codes like 403 on this.

**Comment by hydrian** [comment]
@khorsmann I agree. The HTTP return code should not be a 200. It should be a 4xx code. I'll open a ticket over the pfSense project.

**Comment by hydrian** [comment]
Opened a PFsense forum ticket to see what the possibility of proper return codes being implemented is.
https://forum.pfsense.org/index.php?topic=148079.0
